tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -767,6 +1047,94 @@ Status FragmentMetadata::load_tile_offsets(
   return Status::Ok();
 }
 
+Status FragmentMetadata::load_tile_min_values(
+    const EncryptionKey& encryption_key, std::vector<std::string>&& names) {
+  // Sort 'names' in ascending order of their index. The
```

Comment:
I am a little confused here. We are always storing attribute- and dimension-related data based on the index (i.e., their creation order), not based on their alphanumeric name order. I think we should opt for consistency here and store all the data based on the attribute/dimension index.

Reply:
@stavrospapadopoulos Now that you mention it, so am I! But good point! I blindly followed the pattern of load_tile_offsets. Was there any point in time where you can remember this was the case? If not I can definitely switch the order here for all.

Reply:
No, not really. We always store and load dimension/attribute data based on their index in the schema.

Reply:
@stavrospapadopoulos actually look at the code, it does sort using the built index map. No changes required.